src/main/java/org/candlepin/async/tasks/RefreshPoolsJob.java

```diff
@@ -178,7 +159,6 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
             // Assume that we verified the request in the resource layer:
             this.refresherFactory.getRefresher(this.subAdapter, this.prodAdapter)
                 .setLazyCertificateRegeneration(lazy)
```

Comment:
Do we also want to revert the addition of `.setLazyCertificateRegeneration(lazy)`? It looks like this line was added by commit c9ed71f4b920857564f78e6cb5eeaf8fbf404b49 as well.

Reply:
Commented elsewhere, but this was added as part of a refactor that we're retaining despite reverting the other surrounding logic.